src/lib/commands/remoteFs.ts

```diff
@@ -75,8 +76,18 @@ export const registerCommands = (context: ExtensionContext) => {
         if (!selection || selection === 'Cancel') {
           return;
         }
-        const res = await deleteFile(getPortalId(), filePath);
-        commands.executeCommand('hubspot.remoteFs.refresh');
+        deleteFile(getPortalId(), filePath).then(() => {
+          window.showInformationMessage(`Successfully deleted ${filePath}`);
+          let parentDirectory = dirname(filePath);
+          if (parentDirectory === '.') {
+            parentDirectory = '/';
+          }
+          commands.executeCommand(
+            'hubspot.remoteFs.invalidateCache',
+            parentDirectory
+          );
+          commands.executeCommand('hubspot.remoteFs.refresh');
+        });
```

Comment:
When we delete a file we want to invalidate the cache for its parent directory contents to force it to fetch again, then execute a refresh. The await here was also unnecessarily blocking, we can `then` instead.